src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -57,10 +59,15 @@ object Run  {
   }
 }
 
-case class Run(name: String, pipeline: Pipeline, tag: String) {
+case class Run(jesId: String, pipeline: Pipeline, tag: String) {
+
+  val dataAccess = DataAccess()
```

Comment:
I looked at the DataAccess class. This would be easy to implement as a singleton... however there's a "withDataAccess" method which creates and destroys a new instance of DataAccess for a specific function call. Which implies that there must be some value to not re-using the same one?